### DISCUSSION TITLE
How to Install?

### DISCUSSION THREAD

**Discussion by Cat-o-War** [redeclipse/base#1590] [Help and Support] [answered]
I want to download the bleeding-edge version of Red Eclipse for Windows, but I could not find any game, In the windows.zip file in Deploy repository, I found the game, but it gave me 4 error messages, saying SDL2.dll, libsndfile-1.dll, SDL2_image.dll and OpenAL32.dll were not found. How do I install the game?

**Comment by MirceaKitsune** [comment]
Strange: There used to be a doc page on Github with compile instructions, but it's not in the readme.md of the base repository nor could I see it in the doc directory. I only have the instructions for Linux I'm afraid, but I'm sure someone will clarify and hopefully make the compilation info more obvious in the repo.

The error you're describing sounds like missing libraries however. I'd make sure SDL2 is installed system wide: The engine hasn't yet been ported to SDL3 and I don't know if Windows might have already decided the previous version is too old to maintain.

**Reply by Cat-o-War** [reply]
I installed SDL2, but the game is still giving me the same 4 errors.

**Comment by q009** [comment, marked as answer]
We recommend using the Steam version of the game, which closely follows what's happening on the development version.
Otherwise, you need to clone the `base` repo of the game, with all its submodules. Then you either have to build the game yourself (easiest is using Visual Studio, using the supplied project), or copying the deployed game binaries.

**Reply by Cat-o-War** [reply]
Where are the deployed game binaries?

**Reply by SniperGoth** [reply]
On the redeclipse/deply repo 

https://github.com/redeclipse/deploy
Then you find the binaries there on the master folder

**Comment by qreeves** [comment]
With [Git for Windows](https://git-scm.com/download/win), you can use Git Bash to do:
```
git clone --recurse-submodules https://github.com/redeclipse/base.git redeclipse
cd redeclipse
git submodule update --init
curl -o ../windows.zip https://github.com/redeclipse/deploy/raw/master/master/windows.zip
unzip ../windows.zip -d .
```

**Reply by Cat-o-War** [reply]
The commands did not work, when I did `unzip ../windows.zip -d .` Git Bash gave me the following error: `Archive:  ../windows.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of ../windows.zip or
        ../windows.zip.zip, and cannot find ../windows.zip.ZIP, period.
`

**Reply by Jigoku** [reply]
Looks like it needs "-L" with the curl command, to follow a redirect 👍 
Try changing to this:
```
curl -L -o windows.zip https://github.com/redeclipse/deploy/raw/master/master/windows.zip
```

**Reply by qreeves** [reply]
> Looks like it needs "-L" with the curl command, to follow a redirect

Forgot about the redirect, sorry.
